source/write/update.txt

```diff
@@ -0,0 +1,248 @@
+
+The ``update_one()`` and ``update_many()`` methods accept options to configure the update
+operation. You can pass these options individually as parameters, or you can create a
+``Hash`` object that contains the options and pass it as a parameter.
+If you don't specify any options, the driver performs update
+operations with default settings.
+
+The following table describes the options that you can use to
+configure the update operation:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Option
+     - Description
+
+   * - ``upsert``
+     - | Specifies whether the update operation performs an upsert operation if no 
+         documents match the query filter. For more information, see the :manual:`upsert
+         statement </reference/command/update/#std-label-update-command-upsert>`
+         in the {+mdb-server+} manual.
+       | Defaults to ``false``
+
+   * - ``bypass_document_validation``
+     - | Specifies whether the update operation bypasses document validation. This lets you 
+         update documents that don't meet the schema validation requirements, if any 
+         exist. For more information about schema validation, see :manual:`Schema
+         Validation </core/schema-validation/#schema-validation>` in the MongoDB
+         Server manual.
+       | Defaults to ``false``.
+
+   * - ``collation``
+     - | Specifies the kind of language collation to use when sorting
+         results. For more information, see :manual:`Collation </reference/collation/#std-label-collation>`
+         in the {+mdb-server+} manual.
+
+   * - ``array_filters``
+     - | Provides a list of filters that you specify to select which
+         array elements the update applies to.
+
+   * - ``hint``
+     - | Sets the index to use when matching documents. 
+         For more information, see the :manual:`hint statement </reference/command/update/#std-label-update-command-hint>`
+         in the {+mdb-server+} manual.
+
+   * - ``let``
+     - | Provides a map of parameter names and values to set top-level
+         variables for the operation. Values must be constant or closed
+         expressions that don't reference document fields. For more information,
+         see the :manual:`let statement
+         </reference/command/update/#std-label-update-let-syntax>` in the
+         {+mdb-server+} manual.
+
+Modify Update Example
+`````````````````````
+
+This example creates and passes options to the ``update_one()`` method.
+The example uses the ``$equal`` operation to match documents
+in which the ``name`` field value is ``"Sunrise Pizzeria"``. It then uses the ``$set`` 
+operation to set the ``borough`` field value in the first matching document to
+``"Queens"`` and the ``cuisine`` field value to ``"Italian"``.
+
+Because the ``upsert`` option is set to ``true``, the driver inserts a new document that
+has the fields and values specified in the filter and update document if the query filter
+doesn't match any existing documents.
```

Comment:
S:

```suggestion
Because the ``upsert`` option is set to ``true``, if the query filter
doesn't match any existing documents, the driver inserts a new document that
contains the fields and values in the filter and update documents.
```